server/conf/docker/docker-compose.yml

```diff
@@ -1,12 +1,12 @@
 # This compose file is useful for testing https
 ---
-version: "3"
+version: "3.8"
```

Comment:
__Verification.__ This doesn't break anything, right? _[Version change log](https://docs.docker.com/compose/compose-file/compose-versioning/#version-31) (for people smarter than me about Docker and `docker-compose`)._

Reply:
__Squirrel.__ Do we want to use [`secrets`](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets)?!